presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/optimization/JdbcPlanOptimizerProvider.java

```diff
@@ -32,27 +37,44 @@
     private final StandardFunctionResolution functionResolution;
     private final DeterminismEvaluator determinismEvaluator;
     private final ExpressionOptimizer expressionOptimizer;
+    private final String identifierQuote;
 
+    @Inject
     public JdbcPlanOptimizerProvider(
+            JdbcClient jdbcClient,
             FunctionMetadataManager functionManager,
             StandardFunctionResolution functionResolution,
             DeterminismEvaluator determinismEvaluator,
             ExpressionOptimizer expressionOptimizer)
     {
-        // TODO: Override getConnectorPlanOptimizer in JdbcConnector and add JdbcPlanOptimizer to it
         this.functionManager = requireNonNull(functionManager, "functionManager is null");
         this.functionResolution = requireNonNull(functionResolution, "functionResolution is null");
         this.determinismEvaluator = requireNonNull(determinismEvaluator, "determinismEvaluator is null");
         this.expressionOptimizer = requireNonNull(expressionOptimizer, "expressionOptimizer is null");
+        this.identifierQuote = jdbcClient.getIdentifierQuote();
     }
 
     @Override
     public Set<ConnectorPlanOptimizer> getConnectorPlanOptimizers()
     {
+        RowExpressionToSqlTranslator translator = getRowExpressionToSqlTranslator(
```

Comment:
Pass the result of `buildFunctionTranslator(getFunctionTranslators())` into `JdbcComputePushdown`. The translator is a generic row expr translator. Create `RowExpressionToSqlTranslator` inside `JdbcComputePushdown` as the filter translator.